packages/components/src/tooltip/index.jsx

```diff
@@ -5,29 +5,41 @@ import Popover from '../popover';
 
 import './style.scss';
 
-function Tooltip( props ) {
+function Tooltip( {
+	autoPosition,
+	className,
+	id,
+	isVisible,
+	position = 'top',
+	status,
+	showDelay = 100,
+	showOnMobile = false,
+	hideArrow = false,
+	children,
+	context,
+} ) {
 	const isMobile = useMobileBreakpoint();
 
-	if ( ! props.showOnMobile && isMobile ) {
+	if ( ! showOnMobile && isMobile ) {
 		return null;
 	}
 
-	const classes = clsx( [ 'tooltip', props.className ], {
-		[ `is-${ props.status }` ]: props.status,
+	const classes = clsx( [ 'tooltip', className ], {
+		[ `is-${ status }` ]: status,
 	} );
 
 	return (
 		<Popover
-			autoPosition={ props.autoPosition }
+			autoPosition={ autoPosition }
 			className={ classes }
-			context={ props.context }
-			id={ props.id }
-			isVisible={ props.isVisible }
-			position={ props.position }
-			showDelay={ props.showDelay }
-			hideArrow={ props.hideArrow ?? true }
```

Comment:
@tyxla do you remember why you added the `?? true` bit in #87381? It seems unneeded to me. The `hideArrow` prop is defaulted to `false`, so it cannot have an `undefined` value. Only if `hideArrow` was `null`, then the React `defaultProps` are not applied and the `??` operator would change the value to `true`. Is this "`null` to `true`" behavior really used anywhere?

Reply:
I agree it's unnecessary. I don't see a need for it with the current state of the code. Might have been me leaning on the safe side of things, or not noticing the default prop down there.

Reply:
Good to hear, removing in 8ec54b4645c.